crates/fuel-core/src/combined_database.rs

```diff
@@ -317,7 +317,7 @@ impl CombinedDatabase {
         S: ShutdownListener,
     {
         if let Some(on_chain_height) = self.on_chain().latest_height_from_metadata()? {
-            self.rollback_to(on_chain_height, shutdown_listener)?;
+            let _ = self.rollback_to(on_chain_height, shutdown_listener);
```

Comment:
https://github.com/FuelLabs/fuel-core/issues/2239

Reply:
definitely don't like this

Reply:
Why can't you return the error here? Or at least log it?

Reply:
Okay I see there are a few small cases to consider. It would be nice with a todo link referencing this from the code as well.

Reply:
addressed in 295301924b

Reply:
Thank you! :pray: